packages/discovery-provider/ddl/pg_migrate.sh

```diff
@@ -128,6 +128,7 @@ test_dir() {
 migrate() {
     create_migrations_table
 
+    migrate_dir "utils"
```

Comment:
what's a util vs a function?

Reply:
honestly don't need to move this anymore and don't need this folder now that I'm not using hashids, but this was to get the hashids function to be defined before the migrations run